maintenance/troubleshoot/troubleshooting.md

```diff
@@ -17,11 +17,14 @@ To view logs, use the following command:
 `kubectl logs -n kube-system <pod_name>`
 
 
-To view debug logs on some Calico components, set the `LogSeverityScreen` through the associated environment variable. 
+To view debug logs on some {{site.prodname}} components, set the `LogSeverityScreen` through the associated environment variable. 
 
 
 To report a problem, please [open an issue in GitHub](https://github.com/projectcalico/calico/issues).
 
+### Switching networking modes after initial install
+
+If you make a mistake during initial install configuring your network mode (full {{site.prodname}} or network policy only), there is no clean way to manually back out files to make it work. For example, if you installed the Calico CNI on an EKS cluster (which supports only installing network policy), do not try to back out individual files. Tear down your cluster and try again.  
```

Comment:
@caseydavenport I am not sure if this is indeed the case for switching CNIs. Is this statement valid?

Reply:
My guess is that this statement is maybe too broad / trying to cover too much?

For example, I can imagine a user looking at this section and thinking it applies to any of the following:

- Installing Calico in policy only mode, and then uninstalling Calico (leaving whatever networking intact).
- Switching from one mode to another (e.g., VXLAN to IPIP)
- Switching from one network plugin to another (e.g., Amazon VPC CNI to Calico CNI or vice-versa)

Each of these is really its own scenario, and we don't have good documentation or steps for any of them. I suspect the right solution to that is adding purpose-built docs for each operation so users understand what their options are. Of course, each of these _can_ be solved by tearing down the cluster and starting over again! But, some have other slightly nicer solutions that might not require such drastic measures.

So, I guess my concern is that users might find this section confusing because it isn't specific enough.

Maybe we want the title to be this instead?

```
### Uninstalling Calico on EKS

If you have the luxury to do so, we recommend tearing down your cluster and starting again. However, there are steps you can take....

If you're using policy only mode on top of the Amazon VPC CNI plugin for networking....

If you're using Calico for networking....
```

Reply:
Thanks for inputs. I will close this for now and add docs backlog item.